downloads.md

```diff
@@ -157,6 +176,26 @@ You can add the following dependencies to your `pom.xml` to include Apache Flink
 </dependency>
 ```
 
+### Apache Flink Stateful Functions
+
+You can add the following dependencies to your `pom.xml` to include Apache Flink Stateful Functions in your project.
```

Comment:
Since statefun is new, maybe it makes sense to link to the docs here, to avoid confusion among the regular Flink users who are not following the news?

```suggestion
You can add the following dependencies to your `pom.xml` to include [Apache Flink Stateful Functions](link to docs?] in your project.
```

Reply:
Will do